index.html

```diff
@@ -11,6 +12,103 @@
 </head>
 
 <body>
-<!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
+
+    <div class="container">
+
+        <header>
+
+            <a href="index.html">
+                <img class="logo" src="img/logo.png" alt="Great Idea! Company logo."> </a>
+                
+          <nav>
+            <a href="services.html">Services</a>
+            <a href="#">Product</a>
+            <a href="#">Vision</a>
+            <a href="#">Features</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+          </nav>
+          
+        </header>
+      
+        <div class="services-content">
+      
+          <img class="services-header-img" src="img/services-header.jpg" alt="Our services header image">
```

Comment:
the relative path for this img is img/header-img.png. It can be found in the img folder